### ISSUE TITLE
[Programmatic Access - Foundry Local - Model card]: Labels are not associated with its 'Copy ID' buttons under 'Model card' page.

### ISSUE THREAD

**Issue by chandini1864** [microsoft/foundry-local#1019]
### Describe the issue

[A11y Agency plugin ](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fvigilant-adventure-v9qpqwn.pages.github.io%2fall%2f%23plugins%2fa11y?q%3Da11y&data=05%7c02%7cv-cmadda%40microsoft.com%7cc809251b79d74200ecc908decb9453c2%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639172035135569684%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=5v%2BYGmPOuGLwcQ4QzpTYDdOQmR2PxYEy1h5Z1mZ2ykI%3D&reserved=0)is available to help with accessibility bug remediation.
The required tag has already been added: agency:agent=a11y:a11y-agent@company. Assign the bug to Agency to use it.
Please try it and share feedback on whether the remediation guidance was useful at [CAIA11yAgency@microsoft.com](mailto:CAIA11yAgency@microsoft.com).

**User Experience:**
Screen reader users may not hear the label when they focus a form field, preventing them from readily knowing the field’s purpose. They must explore surrounding text to find context.

Note: User credentials should NOT be included in the bug.

**Actual Result:**
Labels 'parakeet-tdt-0.6b-v2-cuda-gpu:1' and 'parakeet-tdt-0.6b-v2-generic-cpu:1' are not associated with its 'Copy ID' buttons under 'Model card' page.

**Expected Result:**
Labels 'parakeet-tdt-0.6b-v2-cuda-gpu:1' and 'parakeet-tdt-0.6b-v2-generic-cpu:1' should be associated with its 'Copy ID' buttons under 'Model card' page.

**How to Fix:**
Use matching the input’s id. When a native label cannot be used, you can use aria-labelledby to reference the visible label text.

### To reproduce

1. Open the URL "https://www.foundrylocal.ai/" and login with valid credentials.
2. Foundry local homepage opens. Select 'Models' from header.
3. Navigate to any model and activate it, model card opens.
4. Navigate to copy ID buttons.
5. Verify whether labels are associated with its copy id buttons or not.

### Platform and architecture

Windows X64

### OS Version

Windows 25H2 (OS Build 26200.8893)

### Installation type

Released package/binary

### Foundry Local version

https://www.foundrylocal.ai/

### API or surface area

CLI

### Hardware acceleration/backend

Other / Unknown

<img width="1135" height="533" alt="Image" src="https://github.com/user-attachments/assets/66ce4d0b-5a32-4829-ad60-065147825f31" />

**Comment by chandini1864** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#Benchmark;#GH-HCL-FoundryLocal-Aug2026;#MAS1.3.1;#Win11-Edge(Chromium);#agency:agent=a11y:a11y-agent@company;#a11ydb-label-not-associated;

**Comment by prathikr** [comment]
@chandini1864 this issue is fixed. Re-assigning this issue back to you to re-run accessibility analysis.
